ui/store/actions.ts

```diff
@@ -1351,6 +1351,28 @@ export function markNotificationsAsRead(
     await forceUpdateMetamaskState(dispatch);
   };
 }
+
+export function setDesktopEnabled(desktopEnabled: boolean) {
+  return async () => {
+    try {
+      await submitRequestToBackground('setDesktopEnabled', [desktopEnabled]);
+    } catch (error) {
+      log.error(error);
+    }
+  };
+}
+
+export async function generateOtp() {
```

Comment:
how about
```suggestion
export async function generateDesktopOtp() {
```

Reply:
Done, renamed the MetaMask controller method too for consistency.